### PR TITLE
darkhttpd: add service, fix license

### PR DESCRIPTION
**Pull request by prspkt**

### CODE REVIEW

File: srcpkgs/darkhttpd/template [hunk 1]
```diff
@@ -1,19 +1,25 @@
...
-distfiles="$homepage/$pkgname-$version.tar.bz2"
+license="ISC"
+homepage="https://unix4lyfe.org/darkhttpd/"
+distfiles="https://unix4lyfe.org/darkhttpd/darkhttpd-${version}.tar.bz2"
 checksum=a50417b622b32b5f421b3132cb94ebeff04f02c5fb87fba2e31147d23de50505
 CFLAGS="$CFLAGS $LDFLAGS"
 
+# Create darkhttpd system user/group
+system_accounts="${pkgname}"
+make_dirs="/srv/www/${pkgname} 0755 ${pkgname} ${pkgname}"
```
Comment (maxice8) [inline srcpkgs/darkhttpd/template]: why not create /var/log/darkhttpd as well ?

File: srcpkgs/darkhttpd/template [hunk 2]
```diff
@@ -1,19 +1,25 @@
...
+
```
Comment (prspkt) [inline srcpkgs/darkhttpd/template]: i couldn't find any style guidelines for system/user account creation, so i don't know if their position within the template is correct.

**Comment by maxice8** [comment]
On Mon, Nov 05, 2018 at 07:58:10PM +0000, prspkt wrote:
> prspkt commented on this pull request.
> 
> 
> 
> >  checksum=a50417b622b32b5f421b3132cb94ebeff04f02c5fb87fba2e31147d23de50505
>  CFLAGS="$CFLAGS $LDFLAGS"
>  
> +# Create darkhttpd system user/group
> +system_accounts="${pkgname}"
> +make_dirs="/srv/www/${pkgname} 0755 ${pkgname} ${pkgname}"
> +
> 
> i couldn't find any style guidelines for system/user account creation, so i don't know if their position within the template is correct.
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/4434#pullrequestreview-171732844

There isn't well defined as far as i remember but it is long since i touched STYLE.7

also the account name should be prefixed with underscore

File: srcpkgs/darkhttpd/template (hunk 1 above)
Comment (prspkt) [reply srcpkgs/darkhttpd/template]: created `/var/log/darkhttpd` directory

**Comment by prspkt** [comment]
@maxice8 account name prefixed with underscore.

File: srcpkgs/darkhttpd/REMOVE [hunk 3]
```diff
@@ -0,0 +1,5 @@
+case ${ACTION} in
+purge)
```
[the remark this answers is not in the document]
Comment (prspkt) [reply srcpkgs/darkhttpd/REMOVE]: @maxice8 removed `REMOVE`

**Pull request merged**